### ISSUE TITLE
Tweet button doesn't show at bottom when creating a new tweet when the width is so small that the popup/window becomes fullscreen 

### ISSUE THREAD

**Issue by Poopooracoocoo** [Bl4Cc4t/GoodTwitter2#223]
**Please check the following before you continue**
- [x] Checked [FAQ](https://github.com/Bl4Cc4t/GoodTwitter2#faq)
- [x] Checked if the issue has not been reported before

**Describe the bug**
What sites does it affect?
Tweet button doesn't show at bottom when creating a new tweet when the width is so small that the popup/window becomes fullscreen 

**Screenshots**
For display issues, adding screenshots or even videos of your screen is recommended.
![image](https://user-images.githubusercontent.com/51363864/99197419-8444e100-27e6-11eb-8e95-5e345a8a60fc.png)

**Your configuation**
 - Browser: Chrome 86
 - Userscript Manager: ViolentMonkey v2.12.7 (lmk if violentmonkey isn't good)

**Comment by Bl4Cc4t** [comment]
I'm sorry I took so long to respond.
The tweet button is actually located in the top right corner, under the navbar. I'll just hide that bar, should be an easy fix!

**Comment by Poopooracoocoo** [comment]
It's not a problem. I'm glad you replied, even if it's a bit late. Thank you for fixing it! I really appreciate your continued work on GoodTwitter 2. It makes Twitter so much better. Auto refresh got on my nerves. <details><summary>i am weird</summary>
I'm sad that Twitter isn't what it used to be, both as a platform and as a community. Twitter killed third party apps and services and I don't fit into a community anymore. My algorithm is shit and not relevant, and neither are the people I follow. Social media is about people and I don't follow or see people I want to hear from. It doesn't help that social media is becoming more closed: see twitter restricting replies and see instagram removing the ability to see other's likes. And then there are so many people that want money, which these platforms like. It's hard to find genuine cool people. It's not just the algorithms, platform changes, and my past usage of Twitter but it's also just me. and my lack of friends. and my lack of interests
</details>

Do you think that Tampermonkey or Violentmonkey is better?

**Comment by Bl4Cc4t** [comment]
> It's not a problem. I'm glad you replied, even if it's a bit late. Thank you for fixing it! I really appreciate your continued work on GoodTwitter 2. It makes Twitter so much better.

Thanks!

> Auto refresh got on my nerves.

Disable auto refresh works for you? Interesting, it should be pretty broken... I would really like to improve that part but it's really hard to do and I have no idea how to accomplish it.

> i am weird

<details>
<summary>are you weird?</weird>
Although this is off topic, I agree. I only use twitter for looking at adult content and I get so little of that in my actual timeline, it is really annoying. I follow
</details>

**Comment by Bl4Cc4t** [comment]
> It's not a problem. I'm glad you replied, even if it's a bit late. Thank you for fixing it! I really appreciate your continued work on GoodTwitter 2. It makes Twitter so much better.

Thanks!

> Auto refresh got on my nerves.

Disable auto refresh works for you? Interesting, it should be pretty broken... I would really like to improve that part but it's really hard to do and I have no idea how to accomplish it.

> i am weird

<details>
<summary>are you weird?</summary>

Although this is very much off topic, I agree. I only use twitter for looking at adult content and I get so little of that in my actual timeline. 

I follow about 2.7k profiles with only about 100 being non adult themed. Yet most of the actual content gets hidden, it is really annoying. Most of the stuff I get is some random crap I don't care about. So my algorithm isn't better either :D

> It's hard to find genuine cool people. It's not just the algorithms, platform changes, and my past usage of Twitter but it's also just me. and my lack of friends. and my lack of interests

Hm, I can't give you an advice with that, sorry. I just see twitter as a very toned down alternative to the old tumblr days where you could freely post anything you want. 

I loved that platform so much but after the exodus happened more than 2 years ago, twitter was the platform most artists I followed went, so I had to stick with this.
</details>

> Do you think that Tampermonkey or Violentmonkey is better?

They are both very similar, it doesn't really make any difference to the normal user which one you use.

For developing this script I prefer TM since it allows the content of loaded external files to be changed. I need this to update the CSS locally and VM sadly does not allow this. But that is a feature not many need to use anyways, so it really doesn't matter much.

A cool thing about VM is that you can see the contents of the internally set values are (TM does not have that). But this is probably also not very interesting for the normal user.

tl;dr: just use what you like best!

**Comment by Poopooracoocoo** [comment]
>>Auto refresh got on my nerves.
>
>Disable auto refresh works for you? Interesting, it should be pretty broken... I would really like to improve that part but it's really hard to do and I have no idea how to accomplish it.

Well it works in the search for me.

<details>

>Although this is very much off topic

That's why I hid it hehe :P

>So my algorithm isn't better either :D

lol. It's really good to know that I'm not the only one!

>I just see twitter as a very toned down alternative to the old tumblr days where you could freely post anything you want.

Dude the Tumblr purge sucked. I mainly used Tumblr for adult content, but the art and meme communities were really vibrant. Tumblr was also pretty nice as a platform as finding people to follow was so freaking easy with its reblog system. 

</details>


Oh and I wish you a merry Christmas and a happy new year!!!

**Comment by Bl4Cc4t** [comment]
Thank you, merry christmas and a happy new year to you too!
